packages/swapper/src/thorchain-utils/memo/assertAndProcessMemo.ts

```diff
@@ -214,6 +214,9 @@ export const assertAndProcessMemo = (memo: string): string => {
 
       return `${_action}:${asset}:${destAddr}:${minOut ?? ''}:${THORCHAIN_AFFILIATE_NAME}:0`
     }
+    case 'pool+': {
+      return memo
+    }
```

Comment:
Do RUNEPool deposits support affiliate name for tracking?

Reply:
Depends:

On go side, it doesn't handle it: https://gitlab.com/thorchain/thornode/-/blob/develop/x/thorchain/handler.go#L307

In the doc, it doesn't say that it supports it: https://dev.thorchain.org/concepts/memos.html#deposit-runepool

But it might accept it, how do we track?